content/english/java-basics/activity-4.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Control Structures"
+description: "Introduce if-else, for, and while loops in Java."
+date: 2020-07-13T00:00:00Z
+weight: 5
+---
+
+### 1. Make a Decision: If-Statement
+Patrick 🐥 is a very polite duck who likes to greet everyone he meets with "Good Morning," "Good Afternoon," or "Good Night" depends on the current time.
+
+![animation of duck Patrick](https://media.giphy.com/media/hpWrLS1RDBd5pwkgjy/giphy.gif) 
+<p style="text-align: center;">(image by National Geographic: giphy.com/natgeochannel)</p>
+
+As we learned before, we can print these statements as below:
+```java
+System.out.println("Good Morning");   // say it between 0:00 - 11:59
+System.out.println("Good Afternoon"); // say it between 12:00 - 19:59
+System.out.println("Good Night");     // say it between 20:00 - 23:59
+```
+
+However, we also know Patrick 🐥 should only decide to say one of these statements depends on the time. 
+
+In other words, we need to write code that makes a decision on what to print depend on a situation.
+
+In Java, we make a decision by using <b>if statement</b> or <b>if...else statement</b>. Read the notes below:
```

Comment:
In Java, we make a decision by either using an `if` statement or an `if`...`else` statement. Read the notes below: